cmd/svcat/main.go

```diff
@@ -171,7 +173,8 @@ func newDescribeCmd(cxt *command.Context) *cobra.Command {
 
 func newInstallCmd(cxt *command.Context) *cobra.Command {
 	cmd := &cobra.Command{
-		Use: "install",
+		Use:   "install",
```

Comment:
The install subcommand is a placeholder for additional commands, e.g. installing service catalog to a cluster. So either we need some super generic text to explain it, or leave it as-is, without a short description so that people rely on the deeper commands (e.g. svcat install plugin --help) for doc.

Reply:
I couldn't think of a better description for now. But leaving it empty is definitely not ok. If empty, it looks odd - as if we forgot to add a description:
```
$ svcat --help
The Kubernetes Service Catalog Command-Line Interface (CLI)

Usage:
  svcat [flags]
  svcat [command]

Available Commands:
  bind        Binds an instance's metadata to a secret, which can then be used by an application to connect to the instance
  completion  Output shell completion code for the specified shell (bash).
  deprovision Deletes an instance of a service
  describe    Show details of a specific resource
  get         List a resource, optionally filtered by name
  install     
  provision   Create a new instance of a service
  sync        Syncs service catalog for a service broker
  touch       Force Service Catalog to reprocess a resource
  unbind      Unbinds an instance. When an instance name is specified, all of its bindings are removed, otherwise use --name to remove a specific binding
  version     Provides the version for the Service Catalog client and server

```

Reply:
Maybe something like "Various commands for installing service catalog & related tools" ?

Reply:
How about "Install service catalog related tools"?

Reply:
OK, I've changed it. When we add the option of installing the service catalog itself, we can reword it.

BTW, I believe it should be "Install Service-Catalog-related tools", right?

Reply:
I am not an editor but if you want to hyphenate it would be "Service Catalog-related" tools which looks odd to me. "Service Catalog" itself isn't hyphenated.